5-appinfra/apps/cymbal-bank/accounts-userservice/envs/development/main.tf

```diff
@@ -15,17 +15,20 @@
  */
 
 locals {
-  env = "development"
+  env                           = "development"
+  app_namespace                 = "transactions-development"
+  app_service_account_name      = "bank-of-anthos"
+  pod_service_account_principal = "principal://iam.googleapis.com/projects/${local.cluster_project_number}/locations/global/workloadIdentityPools/${local.cluster_project_id}.svc.id.goog/subject/ns/${local.app_namespace}/sa/${local.app_service_account_name}"
 }
 
 module "env" {
-  source = "../../../../../modules/env_baseline"
+  source = "../../../../../modules/alloydb-psc-setup"
 
-  env                      = local.env
-  cluster_project_id       = var.cluster_project_id
-  network_project_id       = var.network_project_id
-  cluster_regions          = var.cluster_regions
-  app_project_id           = var.app_project_id
-  network_name             = var.network_name
-  psc_consumer_fwd_rule_ip = var.psc_consumer_fwd_rule_ip
+  env                         = local.env
+  network_project_id          = var.network_project_id
+  cluster_regions             = var.cluster_regions
```

Comment:
Looks like this is just used as `var.cluster_regions[0]` in the actually module, so I would suggest we re-name to db_region (or similar), and just pass through the needed value.